inc/Engine/Optimization/DelayJS/HTML.php

```diff
@@ -203,7 +203,7 @@ private function parse( $html ): string {
 	 *
 	 * @return string
 	 */
-	public function replace_scripts( $matches ): string {
+	public function replace_scripts( $matches ) {
```

Comment:
why was the return typehint removed?

Reply:
Oh that was not intentional, while debugging wanted to return a different datatype other than a string, just forgot to return it back.